.slashrc

```diff
@@ -131,15 +131,15 @@ class MediaPlugin(slash.plugins.PluginInterface):
 
   def _get_ref_addr(self, context):
     path, case = slash.context.test.__slash__.address.split(':')
-    keyctx = filter(lambda c: str(c).startswith("key:"), context)
+    keyctx = list(filter(lambda c: str(c).startswith("key:"), context))
     if len(keyctx):
       key = keyctx[0].lstrip("key:")
     else:
       key = os.path.relpath(path, self.mypath)
     return ':'.join([key, case])
 
   def _set_test_details(self, **kwargs):
-    for k, v in kwargs.iteritems():
+    for k, v in iter(kwargs.items()):
```

Comment:
Is `iter` actually necessary?  I think `kwargs.items()` is enough.  Same everywhere else.

Reply:
yes, kwargs.items works, will do the change